src/test/scripts/generate-random-data.ts

```diff
@@ -32,7 +33,9 @@ const data = new Array(DOCUMENTS_COUNT).fill(null).map((): HomeDocument => {
         // eslint-disable-next-line @typescript-eslint/ban-types
         propertyAreaSquared: hasProperty && hasPropertyAreaSquared ? areaSquared : (null as unknown as undefined),
         // eslint-disable-next-line @typescript-eslint/ban-types
-        propertyAreaSquaredAsString: hasProperty && hasPropertyAreaSquared ? areaSquared.toString() : (null as unknown as undefined)
+        propertyAreaSquaredAsString: hasProperty && hasPropertyAreaSquared ? areaSquared.toString() : (null as unknown as undefined),
```

Comment:
why do we convert `areaSquared` to string?

Reply:
oh okay, it is being added `...AsString` field - noted 🙏

Reply:
yes, I guess that's what you wanted to do 😃